src/deploy/campaign/get-config.ts

```diff
@@ -275,5 +305,5 @@ export const buildCrosschainConfig = () : TZNSCrossConfig => {
     throw new Error(`Unsupported chain: ${srcChainName}!`);
   }
 
-  return crossConfig;
+  return { crosschain: crossConfig, srcChainName };
```

Comment:
why not just use the name of the variable you're returning when you declare it above for simplicity? or change the return variable name to match? seems like a needless thing to have one prop named the return value and one not